frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -275,14 +274,15 @@ export const HorizontalNav = React.memo((props: HorizontalNavProps) => {
 
   const routes = pages.map((p) => {
     const path = `${props.match.path}/${p.path || p.href}`;
-    const render = (params) => {
+    const render = (params: RouteComponentProps) => {
       return (
         <ErrorBoundary FallbackComponent={ErrorBoundaryFallback}>
           <p.component
             {...componentProps}
             {...extraResources}
             {...p.pageData}
-            params={params}
+            {..._.omit(params, 'match')} // match is being pushed as part of componentProps
```

Comment:
@christianvogt Were you suggesting we remove `params` altogether rather than using the `...` rest parameters here?

Reply:
As per our ts definition we will pass `RouteComponentProps` and additional data to the component registered.

Reply:
I'm still not sure if this really addresses @christianvogt's original comment. @christianvogt?

Reply:
One of my comments was referring to the `params: any` type. I wasn't sure what it was and assumed this was a key value pair of query params. Turns out this is actually from react router.

Not sure there's a point in omitting even if it overrides the other value.